rules/false_positives/rancher.yara

```diff
@@ -3,11 +3,11 @@ rule pull_scripts : override {
     curl_chmod_relative_run_tiny = "medium"
     description = "pull-scripts"
   strings:
-    $binary = "BINARY_NAME=\"charts-build-scripts_${OS}_${ARCH}.exe\""
+    $binary_name = "BINARY_NAME=charts-build-scripts"
+    $charts = "charts-build-scripts"
```

Comment:
consider adding `fullword` declaration so that this doesn't match zcharts-build-scripts